core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/Cluster.java

```diff
@@ -425,51 +446,91 @@ public void setConfigStrategy(ConfigStrategy configStrategy) {
     }
 
     public String getCloudbreakAmbariUser() {
-        return cloudbreakAmbariUser.getRaw();
+        return isNotEmpty(getCloudbreakClusterManagerUser()) ? getCloudbreakClusterManagerUser() : cloudbreakAmbariUser.getRaw();
+    }
+
+    public String getCloudbreakClusterManagerUser() {
+        return cloudbreakClusterManagerUser.getRaw();
     }
 
     public String getCloudbreakAmbariUserSecret() {
-        return cloudbreakAmbariUser.getSecret();
+        return isNotEmpty(getCloudbreakClusterManagerUserSecret()) ? getCloudbreakClusterManagerUserSecret() : cloudbreakAmbariUser.getSecret();
```

Comment:
this should go to the `getCloudbreakClusterManagerUserSecret`
eventually, this (and other old) getters will cease
you can even annotate it with @Depracated